### ISSUE TITLE
Screensaver App not detected/working

### ISSUE THREAD

**Issue by CyrTech** [rocksdanister/lively#1238]
I'm running Lively 2.0.1.0 on Windows 10 21H2.
Lively itself is running perfectly using the "Eternal Light" wallpaper (I changed the wallpaper during my testing, but to no avail)

* By <incident> I found the lively screensaver function under wallpaper active --> screensaver. Then I followed the install instructions: 
--> I downloaded the archive from MS Store, unpackt, right-click --> Install.
--> The windows screensaver settings dialog opened and "Lively" was selected. The little preview screen was empty.
--> Clicking on "settings" or "test" resulted in a little flickering of the preview screen. Actually I expected to see a preview or to launch the lively wallpaper.

* I stopped Lively and restarted it again. Got to the wallpaper --> screensaver settings and it still did not detect "Lively.scr" (asks me to install the plugin) and did not let me change to the seperate settings for background and screensaver "Type of screensaver" was inaccessible.

* I rebooted the system, but nothing changed. 
--> Lively works, but the screensaver is not detected nor it is started if the time-out runs out (1min for testing).

* I started "Lively.scr" by double-click from the "c:\windows\system32" folder and it displayed the default background as screensaver. Just as it would be set, because no change to a separate config is possible.

Not sure if I did something wrong, or if the screensaver-app is brocken.

Regards, Joe

**Comment by rocksdanister** [comment]
>Clicking on "settings" or "test" resulted in a little flickering of the preview screen. 

Re-do this and share the logfile:
Goto Lively settings (Gear icon) -> Misc. -> Log File

**Comment by CyrTech** [comment]
Uh, sorry I indeed forgot to attach the log file I took yesterday
[lively_log_20220527_202440.zip](https://github.com/rocksdanister/lively/files/8791286/lively_log_20220527_202440.zip)
.

**Comment by rocksdanister** [comment]
>Clicking on "settings" or "test" resulted in a little flickering of the preview screen.

`settings` is not implemented properly yet in Lively v2.0 and the Lively window is opened.

For `test` Looking at the log(s) I think you are moving the mouse when clicking test and that's stopping the screensaver immediately:
```
2022-05-27 20:06:22.2508|INFO|Lively.Services.ScreensaverService|Starting screensaver..
2022-05-27 20:06:28.5069|INFO|Lively.Services.ScreensaverService|Stopping screensaver..
2022-05-27 20:06:48.3862|INFO|Lively.Services.ScreensaverService|Starting screensaver..
2022-05-27 20:06:50.4281|INFO|Lively.Services.ScreensaverService|Stopping screensaver..
2022-05-27 20:08:30.4863|INFO|Lively.Services.ScreensaverService|Starting screensaver..
2022-05-27 20:08:31.0984|INFO|Lively.Services.ScreensaverService|Stopping screensaver..
2022-05-27 20:08:34.2388|INFO|Lively.Services.ScreensaverService|Starting screensaver..
2022-05-27 20:08:40.0229|INFO|Lively.Services.ScreensaverService|Stopping screensaver..
```

>The windows screensaver settings dialog opened and "Lively" was selected. The little preview screen was empty.

Its currently turned off due to some multimonitor bugs.

>I stopped Lively and restarted it again. Got to the wallpaper --> screensaver settings and it still did not detect "Lively.scr" (asks me to install the plugin)

Its just a notification/reminder; I could find any API for checking currently selected Windows screensaver - ~one option could be to check if `C:\Windows\System32\Lively.scr` file exists but I am concerned of Anti-virus programs behavior detection falsely flagging.~

>did not let me change to the seperate settings for background and screensaver "Type of screensaver" was inaccessible.

My bad, this feature is not implemented yet.. better to hide the UI.

> Lively works, but the screensaver is not detected nor it is started if the time-out runs out (1min for testing).

Windows tells Lively when to show the screensaver, I don't know the exact algorithm Windows uses to detect idle and start screensaver - its not exact, even if you select 1min it can take upto 5min or more to start.

>I started "Lively.scr" by double-click from the "c:\windows\system32" folder and it displayed the default background as screensaver. Just as it would be set, because no change to a separate config is possible.

Opening the .scr file directly is similar to `Test` without installation so looks like everything is working as intended.
